src/commands/games/blackjack.ts

```diff
@@ -27,15 +27,15 @@ const validateBetAmount = (amount: number): string => {
 @ApplyOptions<CommandOptions>({
   aliases: ['blj', 'bj'],
   description: 'Start a Blackjack game to win some Codey coins!',
-  detailedDescription: `**Examples:**\n
-  \`${container.botPrefix}blackjack 100\`\n
```

Comment:
what does removing the newlines actually do?

Reply:
So I actually tried printing out the strings this time and I realized that the fact that a line is written on a different line already means that there's a newline. With the `\n`, that caused double space instead of single space. The extra whitespace at the beginning of a line made some unwanted indents.